app/controllers/spree/admin/base_controller_decorator.rb

```diff
@@ -1,5 +1,5 @@
-Spree::BaseController.class_eval do
+Spree::Admin::BaseController.class_eval do
```

Comment:
I believe the `Spree::Admin::BaseController` inherits `Spree::BaseController`, so this definition probably isn't necessary.

We can probably remove this file in favour of just `spree/base_controller_decorator.rb`